mkdocs.yml

```diff
@@ -1,12 +1,13 @@
 # Project information
 site_name: 'Open Active Management Technology Cloud Toolkit'
+site_url: 'https://open-amt-cloud-toolkit.github.io/docs'
 repo_url: 'https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit'
 repo_name: 'open-amt-cloud-toolkit/open-amt-cloud-toolkit'
 copyright: 'Copyright &copy; 2021 Intel Corporation'
 site_dir: site/1.4
-google_analytics:
-  - 'UA-185750581-1'
-  - 'auto'
+#google_analytics:
+#  - 'UA-185750581-1'
+#  - 'auto'
 # Configuration
 theme:
   name: 'material'
```

Comment:
Add a new property called `gtag: G-00QEDJW2TS` under `theme` and under `analytics`

Reply:
I think I got it right now.